crates/chain/tests/common/tx_template.rs

```diff
@@ -51,25 +52,34 @@ impl TxOutTemplate {
     }
 }
 
+#[allow(dead_code)]
+pub struct TxTemplateEnv<'a, A> {
+    pub tx_graph: TxGraph<A>,
+    pub indexer: SpkTxOutIndex<u32>,
+    pub txid_to_name: HashMap<&'a str, Txid>,
+    pub canonicalization_mods: CanonicalizationParams,
```

Comment:
Nit: This will probably need updating as well.
```suggestion
    pub canonicalization_params: CanonicalizationParams,
```